### PR TITLE
PLF-7255 Share Activities with Multiple documents

### PR DESCRIPTION
**Pull request by boubaker**

### CODE REVIEW

File: component/webui/src/main/resources/locale/social/Webui_en.properties [hunk 1]
```diff
@@ -102,6 +102,8 @@ UIActivity.label.Activity_Can_Be_Deleted=This activity was not found.
 UIActivity.label.Comment=Comment
 UIActivity.label.Cancel=Cancel
 UIActivity.label.Like=Like
+UIActivity.label.Next=Next
+UIActivity.label.Back=Back
```
Comment (thomasdelhomenie) [inline component/webui/src/main/resources/locale/social/Webui_en.properties]: Should be Previous instead of Back IMO.

Comment (boubaker) [inline component/webui/src/main/resources/locale/social/Webui_en.properties]: I 'm ok with that, please comment the spec to change this, see ACTIVITY_21.

Comment (boubaker) [inline component/webui/src/main/resources/locale/social/Webui_en.properties]: @hghorbel was ok about this too, spec modified

Comment (boubaker) [inline component/webui/src/main/resources/locale/social/Webui_en.properties]: done

Comment (thomasdelhomenie) [inline component/webui/src/main/resources/locale/social/Webui_en.properties]: :+1:

**Pull request merged**